doc/nrf/known_issues.rst

```diff
@@ -341,6 +341,22 @@ Immutable bootloader board restrictions
 Build system
 ============
 
+.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0
+
+KRKNWK-7827: Application build system is not aware of the settings partition
+  The application build system is not aware of partitions, including the settings partition, which for big applications can result in application code overlapping with other partitions.
+  As a consequence, writing to overlapping partitions might remove or damage parts of the firmware, which can lead to errors that are difficult to debug.
+
+  **Workaround:** Define and use a code partition.
```

Comment:
Not sure if this is precise enough if we consider the PM fix. Let me explain.

When we use a workaurind with defined:
`CONFIG_PM_SINGLE_IMAGE=y`
`CONFIG_ PM_PARTITION_SIZE_SETTINGS_STORAGE=<settings_value>`

(how to calculate the `settings_value` below, its important.)

In this case the reported flash area for the application is lowered by the settings storage size, e.g.

```
Memory region         Used Size  Region Size  %age Used
           FLASH:      535088 B       488 KB    107.08%
            SRAM:      114159 B       128 KB     87.10%
        IDT_LIST:         168 B         2 KB      8.20
```
Region size for this platform should be 512KB, but `CONFIG_ PM_PARTITION_SIZE_SETTINGS_STORAGE=0x6000` is defined (0x6000 = 24KB).
Wouldn't be clearer to report that this results in shrinking the effective flash memory available for the application or anything similar? It would correspond with the build output nicely IMO.